resources/assets/enderio/lang/ru_RU.lang

```diff
@@ -573,23 +574,23 @@ enderio.gui.inventorypanel.tooltip.sort.count_down.line2=Начиная с на
 enderio.gui.inventorypanel.tooltip.sort.count_up.line1=Сортировать предметы по их количеству
 enderio.gui.inventorypanel.tooltip.sort.count_up.line2=Начиная с одиночным количеством
 enderio.gui.inventorypanel.tooltip.refill.line1=Refills the crafting grid
-enderio.gui.inventorypanel.tooltip.refill.line2=A normal click puts items
-enderio.gui.inventorypanel.tooltip.refill.line3=for one crafting opperations,
-enderio.gui.inventorypanel.tooltip.refill.line4=while a shift click tries to
-enderio.gui.inventorypanel.tooltip.refill.line5=fill the crafting grid.
+enderio.gui.inventorypanel.tooltip.refill.line2=Обычное нажатие поместит предметы
+enderio.gui.inventorypanel.tooltip.refill.line3=для одной операции крафта,
+enderio.gui.inventorypanel.tooltip.refill.line4=в то время как при нажатии SHIFT попытается
+enderio.gui.inventorypanel.tooltip.refill.line5=заполнить сетку крафта.
```

Comment:
puts -> places
operations -> operation

Just those two little changes would improve the readability a lot